src/task/task.service.ts

```diff
@@ -15,23 +15,28 @@ export class TaskService {
 		private taskMapper: TaskMapper
 	) { }
 
-	public addTask(taskDTO: TaskDTO): Promise<InsertResult> {
+	public addTask(taskDTO): Promise<InsertResult> {
```

Comment:
```suggestion
	public addTask(taskDTO: TaskDTO): Promise<InsertResult> {
```

Reply:
i get the following error when i add TaskDTO type to taskDTO parameter <img width="1131" alt="image" src="https://user-images.githubusercontent.com/57131600/184901401-4470d80b-db89-4d7f-9de5-b648ec9d84d5.png">

Reply:
yes, the problem is that .insert() has a Entity param
use .toEntity(...) from mapper